include/util/coordinate.hpp

```diff
@@ -40,7 +40,7 @@ SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 namespace osrm
 {
 
-constexpr const double COORDINATE_PRECISION = 1e6;
+constexpr const double COORDINATE_PRECISION = 1e7;
```

Comment:
Hrm, this is the main change of the PR.
I will revert the line but will keep rounding coordinates to make conversions correct in https://github.com/Project-OSRM/osrm-backend/blob/1e7/unit_tests/util/coordinate_calculation.cpp#L391-L406